.dockerignore

```diff
@@ -0,0 +1,3 @@
+node_modules
+.git
+website
```

Comment:
`docker-compose up` with above `.dockerignore`:

    real	1m27.618s
    user	0m23.492s
    sys	0m9.614s

`docker-compose up` with `.dockerignore` from this PR:

    real	0m12.910s
    user	0m0.856s
    sys	0m0.414s